### ISSUE TITLE
Caesuras in causal loop diagrams

### ISSUE THREAD

**Issue by KevinDCarlson** [ToposInstitute/CatColab#119]
It seems common to include delay effects in causal loops diagrams by crossing an arrow like so: ----//----> . That could be a nice feature to add.

**Comment by epatters** [comment]
"Caesura," a surprisingly accurate word in this context!

Anyway, nice idea. Thinking about the double theory, I suppose that the composing an arrow with a delay with another arrow (delayed or not) should give an arrow with a delay.

**Comment by KevinDCarlson** [comment]
Yeah, I was like "wait, that looks like...." and then that's what it is! Agreed, I guess it's an absorbing element in the monoid of arrow flavors and there should be a delayed negative one too.

**Comment by epatters** [comment]
Additional context for this issue:

- An example of causal loop diagram with a delay: https://sustainabilitymethods.org/index.php/File:Causal_Loop_Delays.png
- An explanation from ["The Systems Thinker"](https://thesystemsthinker.com/guidelines-for-drawing-causal-loop-diagrams/):

> **Significant delays**. Make sure to identify which (if any) links have significant delays relative to the rest of the diagram. Delays are important because they are often the source of imbalances that accumulate in the system. It may help to visualize pressures building up in the system by viewing the delay connection as a relief valve that either opens slowly as pressure builds or opens abruptly when the pressure hits a critical value. An example of this might be a delay between long work hours and burnout: After sustained periods of working 60+ hours per week, a sudden collapse might occur in the form of burnout.
